keras_cv/layers/preprocessing/random_flip_test.py

```diff
@@ -36,6 +36,13 @@ def test_horizontal_flip(self):
             actual_output = layer(inp, training=True)
             self.assertAllClose(expected_output, actual_output)
 
+    def test_flip_ragged(self):
+        images = tf.ragged.stack([tf.ones((512, 512, 3)), tf.ones((1002, 512, 3))])
+        bounding_boxes = tf.ragged.stack([tf.ones((5, 5)), tf.ones((3, 5))])
+        inputs = {"images": images, "bounding_boxes": bounding_boxes}
+        layer = RandomFlip(mode="horizontal", bounding_box_format="xywh")
```

Comment:
Is better to use `rel_` boxes format to stress test more ragged images with different resolution?